Sources/Application/MainCoordinator.swift

```diff
@@ -35,81 +35,80 @@ final class MainCoordinator: NSObject,
     func start() {
         root.delegate = self
         addTabs()
-        showLogin()
-    }
-    
-    func handleUniversalLink(_ url: URL) {
-        switch UniversalLinkQualifier.qualifyOneLoginUniversalLink(url) {
-        case .login:
-            loginCoordinator?.handleUniversalLink(url)
-        case .wallet:
-            walletCoordinator?.walletSDK.deeplink(with: url.absoluteString)
-        case .unknown:
-            return
+        if userStore.validAuthenticatedUser {
```

Comment:
I've been playing with this a little, and wonder if we need this check, as evaluateRevisit performs the same check and also does full login if the check does not pass.

The only difference is unlock window will be displayed on start, until fullLogin is called.  But since the unlock window doesn't differ from the splash screen, there is no perceptible change in behavior I could see, as for some reason, the "Unlock" button does not appear.  This is even if I set a breakpoint and step through the start up process.   And the code execution through to fullLogin() all happens almost instantaneously anyway.
So start could just be
```
    func start() {
        root.delegate = self
        addTabs()
        windowManager.displayUnlockWindow(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
            evaluateRevisit()
        }
        evaluateRevisit()
    }
```
which is a bit cleaner.   Feel free to ignore if I'm missing something :-)

Reply:
Yes, good shout. I was thinking of how I could do the inverse; remove the `if userStore.validAuthenticatedUser` from `evaluateRevisit()` but yeah that sounds good!